src/test/actions/dashboardAction.test.js

```diff
@@ -0,0 +1,59 @@
+import '@babel/polyfill';
```

Comment:
We don't have to import `@babel/polyfill` in all our test files; app is configured to use `transform-runtime`, which provides same features as polyfill.

Reply:
Great. Thanks for pointing that out. I have removed all occurrences of that import.